devito/operations/interpolators.py

```diff
@@ -188,9 +188,16 @@ def _interpolation_indices(self, variables, offset=0, field_offset=0,
                 condition = sympy.And(lb, ub, evaluate=False)
                 mapper[d] = ConditionalDimension(p.name, self.sfunction._sparse_dim,
                                                  condition=condition, indirect=True)
-
+            # Apply mapper to each variable with origin correction before the
```

Comment:
Some boring nitpicks here:

* blank line
* use "subs", not "vmapper" (more conventional?)
* no full stop at the end of the comment
* dimensions -> Dimensions

Reply:
done